unity-client/Assets/ABConverter/Core.cs

```diff
@@ -450,10 +480,14 @@ internal List<AssetPath> DumpImportableAssets(List<AssetPath> assetPaths)
                 {
                     texImporter.crunchedCompression = true;
                     texImporter.textureCompression = TextureImporterCompression.CompressedHQ;
+                    
+                    ReduceTextureSizeIfNeeded(assetPath.hash + "/" + assetPath.hash + Path.GetExtension(assetPath.file), 512);
```

Comment:
we should move the 512 to a constant since this is important information imo

Reply:
Great Idea. Done!